src/examples/java/com/amazonaws/crypto/examples/RawRsaKeyringDecryptExample.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except
+ * in compliance with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.amazonaws.crypto.examples;
+
+import com.amazonaws.encryptionsdk.AwsCrypto;
+import com.amazonaws.encryptionsdk.AwsCryptoResult;
+import com.amazonaws.encryptionsdk.DecryptRequest;
+import com.amazonaws.encryptionsdk.keyrings.Keyring;
+import com.amazonaws.encryptionsdk.keyrings.StandardKeyrings;
+
+import java.security.KeyPair;
+
+/**
+ * <p>
+ * Decrypts data using the Raw RSA Keyring.
+ */
+public class RawRsaKeyringDecryptExample {
```

Comment:
We usually do encrypt before decrypt so there's a ciphertext to decrypt.

Reply:
My thought was to limit the scope of this example in case you are just focused on decryption. It also matches the suggestion from #108 for separate RSA encrypt and decrypt